packages-exp/auth-exp/src/core/user/reload.test.ts

```diff
@@ -168,6 +168,26 @@ describe('core/user/reload', () => {
     expect(auth.persistenceLayer.lastObjectSet).to.eql(user.toJSON());
   });
 
+  it('nulls out properties in passthrough mode', async () => {
```

Comment:
can you assert getIdToken/logout not called?

Reply:
You got it! Did so for `getIdToken()`